### DISCUSSION TITLE
CPU vs GPU?

### DISCUSSION THREAD

**Discussion by daniswan** [vdemichev/DiaNN#627] [Q&A]
I am putting together specs for a new data analysis computer, which we will use for DIA-NN analysis of both timsTOF and Oribtrap data.  Previously I have focused on buildling CPU intensive computers for proteomics data analysis, but I wasn't sure if software like DIA-NN are best suited for CPU, GPU, or if both work well.

**Comment by vdemichev** [comment]
Best/$:
- 16+ cores, high core frequency; 7950X / 7950X3D, 10980XE - like that
- 128Gb RAM
- At least some SSD to store things like spectral libraries, can be small
- GPU is irrelevant
Threadripper 64-core configs are a lot more expensive and might be good for running multiple large analyses in parallel.
